cloud/services/groups/groups_test.go

```diff
@@ -150,6 +150,16 @@ func TestDeleteGroups(t *testing.T) {
 				)
 			},
 		},
+		{
+			name:          "resource group does not exist",
```

Comment:
how is this use case different than the one above?

Reply:
the error is returned from a different step, but good point about the name, will update that